### PR TITLE
Make markdown extensions configurable with Django settings

### PR DESCRIPTION
**Pull request by danihodovic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,8 +22,16 @@ extensions to make it actually useful in Wagtail:
 * Inline Markdown preview using [SimpleMDE](http://nextstepwebs.github.io/simplemde-markdown-editor/)
 
 These are implemented using the `python-markdown` extension interface.
-Currently, adding new extensions isn't possible without modifying the code, but
-that shouldn't be difficult to implement (patches welcome).
+
+You can configure wagtail-markdown with additional extensions using django
```
Comment (zerolab) [inline README.md]:
```suggestion
You can configure wagtail-markdown to use additional Markdown extensions by via the `WAGTAILMARKDOWN_EXTENSIONS` setting.
```

**Comment by danihodovic** [comment]
@zerolab looks better now?

**Comment by StefanUlbrich** [comment]
One neat addition were to add custom filters. In the case of formulas, some extensions add a `<script type="math/tex">` field which gets sanitized. I experimented writing such a filter (see `utils.py` in [this branch](https://github.com/StefanUlbrich/wagtail-markdown/tree/feat/mathjax) of my fork)

**Comment by danihodovic** [comment]
@zerolab could you have a look at this again?

**Comment by adinhodovic** [comment]
Can we merge this @zerolab ?

**Comment by zerolab** [comment]
Thank you for the nudge @adinhodovic, LGTM and merged

**Pull request merged**